app/services/data_scraping.rb

```diff
@@ -0,0 +1,58 @@
+class DataScraping < ApplicationService
```

Comment:
As an improvement, it would be best to name services using verbs (thinks of command objects or use cases). In addition, file names must be suffixed with their type to follow Rails conventions (i.e., `_controller`):

```ruby
class ScrapeGoogleService < ApplicationService
```

or 

```ruby
# Namespacing all scrapers under a Search module
class Search::ScrapeGoogleService < ApplicationService
```